Makefile-docker

```diff
@@ -94,7 +94,7 @@ update_deps_base: ## update the python and node dependencies
 .PHONY: update_deps
 update_deps: update_deps_base ## update the python and node dependencies for development
 	$(PIP_COMMAND) install --progress-bar=off --no-deps --exists-action=w -r requirements/dev.txt
-	npm install $(NPM_ARGS)
+	npm install $(NPM_ARGS) --no-save
```

Comment:
I guess this is harmless because it's just `npm install` without any package specified to install ourself but why would we need it adding?

Reply:
It prevents any changes to package.json. If for some reason running npm install would lead to a change then it's possible for l10n extraction to commit changes to package.json. I ran into this edge case before and that's why I added it. 

This way in order to update you have to actually npm install something, kind of like we need to hashin to update a requirements file.